modules/op-resolver-tasks-step-actions-about.adoc

```diff
@@ -0,0 +1,65 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: CONCEPT
+[id="resolver-tasks-step-actions-about_{context}"]
+
+= About non-versioned and versioned tasks and step actions
+
+The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates the following items:
+
+* `buildah-1-16-0` versioned task
+* `buildah` non-versioned task
+* `git-clone-1-16-0` versioned `StepAction` definition
+* `git-clone` non-versioned `StepAction` definition
+
+Non-versioned and versioned tasks and step actions have the same metadata, behavior, and specifications, including `params`, `workspaces`, and `steps`. However, they behave differently when you disable them or upgrade the Operator.
+
+Before adopting non-versioned or versioned tasks and step actions as a standard in production environments, cluster administrators might consider their advantages and disadvantages.
+
```

Comment:
IDK what the latest guidance is on TOC level, but it might (ha) be nice to have L3 headings above these table sections for scannability. I am not certain where we are landing with the depth of the TOCs on docs.redhat, though, so take my thought with a grain of salt.